templates/participants.html

```diff
@@ -68,3 +79,63 @@ <h1>Wszyscy uczestnicy</h1>
     </article>
 </div>
 {% endblock %}
+
+{% block script %}
+  <script>
+    $(document).ready(() => {
+      const table = $('#participants-table').DataTable({
```

Comment:
I don't like the default settings. Too many important things are not on screen by default. Cover letter, status and comments are more important than e.g. school, matura year or even email.